internal/merkle/memory/memory_merkle_tree.go

```diff
@@ -12,10 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package merkle
```

Comment:
The name of this file can be shortened (no need in `memory_` prefix since it's now in the path).

Reply:
done